subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -9743,6 +9743,22 @@ u32_t ll_terminate_ind_send(u16_t handle, u8_t reason)
 	return 0;
 }
 
+#if defined(CONFIG_BT_CTLR_CONN_RSSI)
+u32_t ll_rssi_get(u16_t handle, u8_t *rssi)
+{
+	struct connection *conn;
+
+	conn = connection_get(handle);
+	if (!conn) {
+		return 1;
```

Comment:
This is part of the LE Controller style fixes Jira issue, will be done as one later on